Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -2192,3 +2209,8 @@ clearDriverFeeWithCreate (personId, merchantId, opCityId) serviceName (fee', mbC
         (_, _) -> Just (0.0, 0.0)
 
     calcPercentage percentage = (percentage * fee') / 100.0
+
+verifyVpaStatus :: (EsqDBFlow m r, MonadFlow m, CacheFlow m r) => (Id SP.Person, Id DM.Merchant, Id DMOC.MerchantOperatingCity) -> m APISuccess
+verifyVpaStatus (personId, _, _) = do
```

Comment:
How driver themselves will verify?

Reply:
ui will call this api after drivers will confirm the shown vpa acc to the conditions